spec/factories/event_schedule.rb

```diff
@@ -0,0 +1,22 @@
+# Read about factories at https://github.com/thoughtbot/factory_girl
+
+FactoryGirl.define do
+  factory :event_schedule do
+    event
+    after(:build) do |event_schedule|
```

Comment:
Uff, there is a lot of logic in the factory. Is that really all necessary to get a valid factory?